crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -243,6 +250,19 @@ fn context_key_from_key(key: &[u8; 32]) -> u64 {
 }
 
 impl RealRibosome {
+    pub fn standard_usage_meter() -> Arc<Counter<u64>> {
+        meter_with_version(
+            "hc.ribosome.wasm",
```

Comment:
Can new metrics go into the docs in the `holochain_metrics` crate docs please? 

It would also be good to provide a dashboard, or a change to an existing dashboard to display this metric